nameable.rb

```
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name()
+    puts 'NotImplementedError'
+  end
```

- [ ] This method should raise an exception, not print one.

  ```
  def correct_name
    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
  end
  ```